src/locales/cy/translation.json

```diff
@@ -15,13 +15,12 @@
       "signOut": "Allgofnodi",
       "phaseBanner": {
         "tag": "beta",
-        "content": "Mae hwn yn wasanaeth newydd – bydd eich <a class=\"govuk-link\" rel=\"noopener\" target=\"_blank\" href=\"https://signin.account.gov.uk/contact-us\">adborth (agor mewn tab newydd)</a> yn ein helpu i’w wella."
+        "content": "Mae hwn yn wasanaeth newydd – bydd eich <a class=\"govuk-link\" rel=\"noopener\" target=\"_blank\" href=\"{{url}}\">adborth (agor mewn tab newydd)</a> yn ein helpu i’w wella."
       }
     },
     "shared": {
       "govUKHomepageButtonHref": "https://www.gov.uk/",
       "govUKHomepageButtonText": "Ewch i hafan GOV.UK",
-      "contactLinkHref": "https://signin.account.gov.uk/contact-us",
```

Comment:
This is the same in English and Welsh so it doesn't need to be in the translation file